kubernetes/loculus/templates/_common-metadata.tpl

```diff
@@ -370,12 +370,13 @@ organisms:
   {{ $key }}:
     {{- with $instance.schema }}
     {{- $nucleotideSequences := .nucleotideSequences | default (list "main")}}
-    ingest: {{- $instance.ingest.configFile | toYaml | nindent 8 }}
+    enaDeposition: {{- $instance.enaDeposition.configFile | toYaml | nindent 6 }}
+      segment_names: {{- $nucleotideSequences | toYaml | nindent 8 }}
```

Comment:
Are we using `segment_names` anywhere? Can't see the change anywhere else in the code, maybe let's revert for now to keep the changeset smaller and do the rename later

Reply:
Totally we aren't using segment_names yet - would be good to refactor the code to use this though to not rely on single segmented pathogens having a segment called main - but adding this line atm has absolutely no effect on the code.

Reply:
I removed but would be good to add in a later PR

Reply:
Thanks for explaining we don't use it right now!